### ISSUE TITLE
iOS: In silent mode no vibrate

### ISSUE THREAD

**Issue by davidellolio** [phonegap/phonegap-plugin-push#1509]
### Expected Behaviour
When switch on the silent mode in Ios devices the vibration work and push is displayed like whatsapp.

### Actual Behaviour
When switch on the silent mode in Ios devices the vibration not work and push is diplayed.

### Reproduce Scenario 

The plugin work fine. why switch on the silent mode in Ios devices the vibration not work when receive the push notification?

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)

iOS 10.1

#### Cordova CLI version and cordova platform version
6.4.0
 4.3.0

#### Plugin version

 1.9.0

**Comment by macdonst** [comment]
@davidellolio did you turn off "vibrate on silent" on your test device?

**Comment by davidellolio** [comment]
No, The vibrate is on in silent mode . I receive push notification in WhatsApp and the device vibrate. When receive the notification with this plugin don't vibrate
